tests/wpt/web-platform-tests/url/urlsearchparams-foreach.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<link rel="help" href="http://url.spec.whatwg.org/#dom-urlsearchparams-has">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+test(function() {
+    var params = new URLSearchParams('a=1&b=2&c=3');
+    for(var URLSearchParam in params){
```

Comment:
This test doesn't actually verify that the keys are present. I don't think `for in` works on iterators this way, either, since Firefox didn't show me any of those keys when I tried it.